packages/typespec-azure-resource-manager/src/rules/arm-resource-invalid-action-verb.ts

```diff
@@ -2,7 +2,7 @@ import { Operation, createRule } from "@typespec/compiler";
 import { getOperationVerb } from "@typespec/http";
 
 import { getActionDetails } from "@typespec/rest";
-import { isInternalTypeSpec, isSourceOperationResourceManagerInternal } from "./utils.js";
+import { isInternalTypeSpec } from "./utils.js";
 
 export const invalidActionVerbRule = createRule({
   name: "arm-resource-invalid-action-verb",
```

Comment:
while you are here would you mind adding teh documentationUrl and doc page for this rule(check `no-enum.ts` for example)